crm-project/src/main/java/CRM/service/StatusService.java

```diff
@@ -15,4 +17,16 @@ public class StatusService implements AttributeService {
     public int update(long id, Attribute object) {
         return 0;
     }
+
+    /**
+     * This function persists a new Status to the database by calling the save function in the StatusRepository class.
+     * @param status The status object to be persisted.
+     * @return The persisted status object.
+     */
+    @Override
+    public Status create(Attribute status) {
+        if(statusRepository.existsByBoardAndNameLike(status.getBoard(), status.getName()))
+            throw new NonUniqueObjectException("Status with the same name already exists in this board", status.getId(), "Status");
```

Comment:
When throwing exceptions, we have a dedicated enum class for error messages (ExceptionMessages).
Add your own message there if it doesn't exist, and then post it here.